packages/node_modules/@webex/plugin-meetings/test/unit/spec/member/util.js

```diff
@@ -0,0 +1,44 @@
+import {assert} from '@webex/test-helper-chai';
+
+import MemberUtil from '@webex/plugin-meetings/src/member/util';
+
+
+describe('isHandRaised', () => {
+  it('throws error when there is no participant', () => {
+    assert.throws(() => {
+      MemberUtil.isHandRaised();
+    }, 'Raise hand could not be processed, participant is undefined.');
+  });
+
+  it('return false when hand is not there in controls', () => {
+    const participant = {
+      controls: {}
+    };
```

Comment:
suggestion: A test where `controls` is missing (i.e. `const participant = {}') would be good for completeness (the implementation has the optional chaining operator on `controls`).